src/main/resources/openapi.json

```diff
@@ -1,314 +1,1279 @@
+                        "type": "integer"
+                    },
+                    "message": {
+                        "type": "string"
+                    }
+                },
+                "example": "{\n    \"error_code\": 404,\n    \"message\": \"resource not found\"\n}"
+            },
+            "Partitions": {
+                "title": "Partitions",
+                "type": "object",
+                "properties": {
+                    "partitions": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/components/schemas/Partition"
+                        }
+                    }
+                },
+                "example": {
+                    "partitions": [
+                        {
+                            "topic": "topic",
+                            "partition": 0
+                        },
+                        {
+                            "topic": "topic",
+                            "partition": 1
+                        }
+                    ]
+                }
+            }
         }
-      },
-      "parameters": [
+    },
+    "tags": [
         {
-          "name": "groupid",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
+            "name": "Consumers",
+            "description": "Consumer operations to create consumers in your Kafka cluster and perform common actions, such as subscribing to topics, retrieving processed records, and committing offsets."
         },
         {
-          "name": "name",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
-        }
-      ]
-    },
-    "/consumers/{groupid}/instances/{name}/assignments": {
-      "post": {
-        "operationId": "assign",
-        "responses": {
-          "200": {
-            "description": "OK"
-          }
-        }
-      },
-      "parameters": [
+            "name": "Topics",
+            "description": "Topic operations to send messages to a specified topic or topic partition. You can include message keys in requests."
+        },
         {
-          "name": "groupid",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
+            "name": "Seek",
+            "description": "Seek operations that enable a consumer to begin receiving messages from a given offset position."
         },
         {
-          "name": "name",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
+            "name": "Producer",
+            "description": "Producer operations to send records to a specified topic or topic partition."
         }
-      ]
+    ],
+    "externalDocs": {
+        "description": "Find out more about Swagger",
+        "url": "http://swagger.io"
```

Comment:
Is this right, or should it point to Strimzi's docs for the bridge?

Reply:
yeah it should be a link to the Strimzi website where we have doc just for the bridge.

Reply:
externalDocs:
    description: Find out more about the Strimzi HTTP Bridge
    url: 'https://strimzi.io/documentation/'